scripts/executeBoldUpgrade.ts

```diff
@@ -315,32 +369,44 @@ async function checkOldRollup(params: VerificationParams) {
   ) {
     throw new Error('Old rollup was not upgraded')
   }
+
+  // using the reader factory here for typing
+  const rollupReaderContract = RollupReader__factory.connect(config.contracts.rollup, l1Rpc)
```

Comment:
why does this rollupreader still exist? i thought we were getting rid of it

Reply:
it is in the #268 PR, I don't want to mix the contract change with this PR but that will be merged after this one